packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models.test.ts

```diff
@@ -43,7 +43,7 @@ describe('CDK GraphQL Transformer', () => {
 
   const [username, password, identifier] = generator.generateMultiple(3);
 
-  const region = process.env.AWS_REGION ?? 'us-east-1';
+  const region = process.env.CLI_REGION ?? 'us-west-2';
```

Comment:
The test was failing because `AWS_REGION` was not set and the DB and vpc were being created in `us-east-1`. Then the graphql api was created in another region (whatever was specified in `CLI_REGION`).

* `CLI_REGION` is set on CodeBuild, not `AWS_REGION`.
* `us-west-2` is the default region for this backend https://github.com/aws-amplify/amplify-category-api/blob/main/packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-models/app.ts#L38.